lib/mix/tasks/nimble.phx.gen.template.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Mix.Tasks.Nimble.Phx.Gen.Template do
+  @shortdoc "Generates Nimble's Phoenix template"
+
+  use Mix.Task
+
+  alias Nimble.Phx.Gen.Template.{Project, Template}
+
+  @version Mix.Project.config()[:version]
+  @switches [api: :boolean, web: :boolean]
```

Comment:
Not sure about the naming `switches` 🤔 What do you think of `variants` since it's the actual name used in the file structure?